js-pkg/sdk/src/main.ts

```diff
@@ -187,3 +199,19 @@ export async function createDoc(
   const manager = new DocumentManager(connectionString)
   return await manager.createDoc(docId)
 }
+
+/**
+ * A convenience wrapper around {@link DocumentManager.createDocWithContent} for creating a new document
+ * with initial content.
+ *
+ * @param connectionString A connection string (starting with `ys://` or `yss://`) referring to a y-sweet server.
+ * @param update A Yjs update byte string representing the initial content.
+ * @returns A {@link DocCreationResult} object containing the ID of the created document.
+ */
+export async function createDocWithContent(
+  connectionString: string,
+  update: Uint8Array,
+): Promise<DocCreationResult> {
+  const manager = new DocumentManager(connectionString)
+  return await manager.createDocWithContent(update)
+}
```

Comment:
I'm actually tempted to deprecate these functions that sit outside of the `DocumentManager`, so I'm ok with just making this a function on the document manager.